infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/ClusterDescriptor.java

```diff
@@ -54,11 +54,12 @@ public ClusterDescriptor(ClusterDescriptor info, ClusterRole roleType) {
         }
     }
 
-    public ClusterDescriptor(String clusterId, ClusterRole roleType, int corfuPort) {
+    public ClusterDescriptor(String clusterId, ClusterRole roleType, int corfuPort,
```

Comment:
can be replaced with @AllArgsConstructor ?

Reply:
done